### ISSUE TITLE
Alt+Tab integration doesn't work for administrator owned processes

### ISSUE THREAD

**Issue by kvakulo** [kvakulo/Switcheroo#33]
1. Turn on Alt+Tab integration in Switcheroo
2. Open e.g. `cmd` with `Run as administrator`
3. Press `Alt+Tab` while having the cmd window in focus
4. The usual Windows Alt+Tab functionality appears, and not Switcheroo

Is the solution to run Switcheroo as administrator? Or is there another way?

**Comment by kvakulo** [comment]
It is possible by giving Switcheroo `uiAccess="true"` in the application manifest and signing switcheroo.exe with a valid certificate.

Next step is to look into the price and procedure of getting a signing certificate for this purpose.

Resource: http://blogs.techsmith.com/inside-techsmith/devcorner-debug-uiaccess/

**Comment by kvakulo** [comment]
"Search Panel" and "Start menu" shows up when running with `uiAccess="true":

![2015-06-23_13h24_51](https://cloud.githubusercontent.com/assets/359614/8305021/5bb8075c-19ab-11e5-8bb6-6866b4b28553.png)

**Comment by kvakulo** [comment]
At least on Windows 8.1 those windows can be filtered by adding the following to the alt-tab window check:

```
        private bool IsWindowsSearchPane()
        {
            return Title == "Search Pane" && ClassName == "SearchPane";
        }

        private bool IsWindowsSearchMenu()
        {
            return Title == "Start menu" && ClassName == "ImmersiveLauncher";
        }
```

Need to look into how it behaves on other Windows versions.

About the code signing certificate: 95$ from here http://codesigning.ksoftware.net/#read_more

**Comment by guilhas** [comment]
Hello,

Also if
 - Using Alt-Tab shortcut
 - With Admin process windows focused
It will bring the windows default window switcher not Switcheroo.

Happens the same with VistaSwitcher.
http://www.ntwind.com/software/vistaswitcher.html

Regards,
Lino

**Comment by mczaickipl** [comment]
Hi,

Any progress on it? I have Total Commander always in elevated mode, so this one is making very hard to use the app

**Comment by kvakulo** [comment]
Released in v0.10.0-beta.1
